timeseries/setup.py

```diff
@@ -30,7 +30,7 @@
     "torch",  # version range defined in `core/_setup_utils.py`
     "statsmodels>=0.13.0,<0.15",
     "gluonts>=0.13.1,<0.14",
-    "pytorch-lightning>=1.7.4,<1.10.0",
+    "pytorch-lightning>=2.0.0,<2.1",
```

Comment:
Changes to `timeseries` look good to me, I validated that everything works as expected locally with `pytorch-lightning` v2.0.

Feel free to merge once the changes to `multimodal` are approved.